browser/src/control/Control.StatusBar.js

```diff
@@ -405,8 +405,13 @@ L.Control.StatusBar = L.Control.extend({
 
 		this._updateToolbarsVisibility();
 
-		if (isReadOnly)
+		if (isReadOnly) {
 			statusbar.disable('LanguageStatus');
+			statusbar.hide('InsertMode');
+			statusbar.hide('break6');
+			statusbar.hide('StatusSelectionMode');
+			statusbar.hide('break7');
```

Comment:
This could go into `_updateToolbarsVisibility` function -> will be applied on any status change, not only when created.

Reply:
(with else block for showing elements)

Reply:
done, thanks